components/layout/Logo.js

```diff
@@ -1,3 +1,16 @@
-const Logo = () => <div>Site logo here</div>;
+import Image from 'next/image'
 
-export default Logo;
+const Logo = () => {
+    return (
+        <>
+            <Image 
+            src="/../public/images/Whatodogrey.svg"
+            alt="Logos"
+            width="150"
```

Comment:
We will need to use responsive CSS (with external stylesheet or styled components) to style the dimensions of the logo later.